### PR TITLE
Fix-up minor editorial issues

### PR DESCRIPTION
**Pull request by steven-bellock**
Minor edits I made while reading the specification.

Signed-off-by: Steven Bellock <sbellock@nvidia.com>

### CODE REVIEW

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -914,8 +914,8 @@ $version-scheme /= int / text
 
 The following details the security version number (`svn`) and the minimum security version number (`min-svn`) statements.
 A security version number is used to track changes to an object (e.g., a secure enclave, a boot loader executable, a configuration file, etc.) that are security relevant.
-Rollback of a security relevant change is considered to be an attack vector, as such, security version numbers can't be decremented.
-If a security relevant flaw is discovered in the Target Environment and subsequently fiexed, the `svn` value is typically incremented.
+Rollback of a security relevant change is considered to be an attack vector and as such, security version numbers cannot be decremented.
```
Comment (deeglaze) [inline draft-ietf-rats-corim.md:917]:
```suggestion
Rollback of a security relevant change is considered to be an attack vector; as such, security version numbers MUST NOT be decremented.
```

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -914,8 +914,8 @@ $version-scheme /= int / text
...
+If a security relevant flaw is discovered in the Target Environment and subsequently fixed, the `svn` value is typically incremented.
```
Comment (deeglaze) [inline draft-ietf-rats-corim.md:918]:
```suggestion
If a security relevant flaw is discovered in the Target Environment and is subsequently fixed, incrementing the `svn` value is RECOMMENDED.
```

File: draft-ietf-rats-corim.md (hunk 1 above)
Comment (steven-bellock) [reply draft-ietf-rats-corim.md]: I updated the editorial suggestions, but I do not want to add or remove normative wording in this pull request.

File: draft-ietf-rats-corim.md (hunk 2 above)
Comment (steven-bellock) [reply draft-ietf-rats-corim.md]: I updated the editorial suggestions, but I do not want to add or remove normative wording in this pull request.

**Review by deeglaze — APPROVED** [review]

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Review by thomas-fossati — APPROVED** [review]
Thanks @steven-bellock !

**Review by nedmsmith — APPROVED** [review]
LGTM

**Comment by nedmsmith** [comment]
The .ghpages workflow needs to be applied to get the build to succeed.

**Comment by steven-bellock** [comment]
> The .ghpages workflow needs to be applied to get the build to succeed.

Done.

**Pull request merged**